test/JDBC/input/transactions/BABEL_4411.sql

```diff
@@ -0,0 +1,212 @@
+-- Escape hatch disabled
+
+SET TRANSACTION ISOLATION LEVEL READ COMMITTED
+GO
+SELECT current_setting('transaction_isolation')
+SELECT current_setting('default_transaction_isolation')
+SELECT set_config('babelfishpg_tsql.escape_hatch_set_transaction_isolation_level', 'strict', false)
```

Comment:
can we set this escape hatch using TSQL set syntax? if yes, please use TSQL syntax